code/modules/mob/living/living_defense.dm

```diff
@@ -132,7 +146,7 @@
 		return
 	shake_animation(damage)
 
-
+ // return PROJECTILE_CONTINUE if bullet should continue flying
```

Comment:
Where is that return PROJECTILE_CONTINUE  in `bullet_act`? I'm only seeing some return TRUE

Reply:
![image](https://user-images.githubusercontent.com/65828539/193127870-a22e4ee8-703a-4b40-a790-2fd8ebd9888d.png)
![image](https://user-images.githubusercontent.com/65828539/193127858-a391dc5e-7138-4910-b828-6f05f3bcd6df.png)

Those are only used situationally.